.env

```diff
@@ -12,22 +12,24 @@ CLUSTER_DOMAIN_NAME="celo-networks-dev"
 TESTNET_PROJECT_NAME="celo-testnet"
 
 BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
-BLOCKSCOUT_DOCKER_IMAGE_TAG="5fba4843b3e78b5ab75d01766214cb24c6a40649"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="8be64adeccbb8f67dea3b6653600021577e544c5"
 BLOCKSCOUT_WEB_REPLICAS=3
 BLOCKSCOUT_DB_SUFFIX=
 
 ETHSTATS_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/ethstats"
-ETHSTATS_DOCKER_IMAGE_TAG="0ffe524c625ea59e4492dc92c2e638689c36e4b0"
+ETHSTATS_DOCKER_IMAGE_TAG="49fe674cb57f5d11174f4608f7469b03a69ebea5"
+ETHSTATS_TRUSTED_ENODES="ec86feaa5738d806cbe5431cfef97ce147c6ca4efa815f5684008c2f9455d8d39a8540e3fa19c151b3c9f5db89e86cb2565a05a4dbfb2d13f30834a01297e964,0eec0157fbe47a02899d5a5acb2c7309a9ffe4a02e97dcf841e9b809fb7608fc7b1aecf58ee4a8ce0b6ff26e3b7343b7373f553fc83b43596f8f58911afef9ba,b81a149fc0dbf96e9b5e36554896afabe442d318e1861d949e11fb9cbd05ae67236dc08904601a4033cb66e253f37fa335c6fa3d0ff2aa247b3134a904a90df4,cefb1c211db59c17d587cf7f14cb62bb7e46693f44e5d571d8224cdee72819ed22b1a28b13a64efab3a97493669adcae69b8cc93e52c9a8202d3a9fedfbe4a9e,ac1b214fcbb1ab6374ed8a17380199aea3e1088afbb67bc1cd5ee2e7f5b986f3255f8531ae01d5dbf0c248fbe4137204b180c81a4436ee4ac808a659b5621cc7,59cbf3ef26eb6e1c82694c22cf8f10657d30f746eefc3ae3539f3f938b8894fcdce062bc3b56b2ec996d9f603574bfe6396918a40bdf7b74a1fa3c10e4342394"
```

Comment:
Rather than putting these in the .env files I think it'd be best to generate these inside celotool and without directly pulling them from kubernetes pods so that this would work for VM testnets as well.

So when we set the helm parameters for ethstats, we could generate the enodes of all `envVar.TX_NODES` and `envVar.VALIDATORS` similar to how we generate the bootnode enode here: https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/vm-testnet-utils.ts#L390

We generate the keys in Kubernetes & VM testnets the same way (ex: https://github.com/celo-org/celo-monorepo/blob/master/packages/helm-charts/testnet/templates/validators.statefulset.yaml#L61) so this would work for both types of testnets